etna/lib/commands.rb

```diff
@@ -24,10 +24,12 @@ def dispatch_to_subcommand(cmd = 'help', *args, environment: nil, **kwds)
   def environment
     if @environment
       @environment
-    elsif @config && @config.keys.length == 1
+    elsif @config && @config.is_a?(Hash) && @config.keys.length == 1
       @config.keys.last.to_sym
+    elsif @config && @config.is_a?(Hash) && @config.keys.length > 1
+      raise "You have multiple environments configured, please specify your environment by adding --environment #{@config.keys.join("|")}"
```

Comment:
If you have multiple environments, require that it be set with `--environment` on any command that requires it.